Week3/create-data.js

```diff
@@ -0,0 +1,31 @@
+const mysql = require('mysql');
+const util = require('util');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world'
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function createData() {
+  let sql = "CREATE DATABASE IF NOT EXISTS todo_data"
+  let use_sql = "USE todo_data"
+  let users = "CREATE TABLE IF NOT EXISTS Users(Id varchar(100) NOT NULL PRIMARY KEY UNIQUE, FirstName varchar(50) NOT NULL, LastName varchar(50))"
```

Comment:
Try to look into AUTOINCREMENT: https://dev.mysql.com/doc/refman/8.0/en/example-auto-increment.html

If you check the link, you see that with auto increment you don't need to provide an id as it is generated by MySQL itself.